build/mac/BUILD.gn

```diff
@@ -8,6 +8,16 @@ declare_args() {
 _packaging_dir = "$root_out_dir/$brave_product_name Packaging"
 keychain_db = getenv("HOME") + "/Library/Keychains/login.${mac_signing_keychain}-db"
 
+_target_app_name = "Brave-Browser"
```

Comment:
does it make sense to create vars for "Brave-Browser" and "-Beta", etc.. in a gni file so it can be shared across platforms and stay consistent? I think the linux one I just reviewed could be combined as well

Reply:
even if they use slightly different names, I think it would be good to have the logic for building the name in the same place

Reply:
maybe that same gni file should create defines that we can use in the c++ code?

Reply:
Deleted and logic in `//brave/build/config.gni` is used. PTAL.